### DISCUSSION TITLE
how did you got the data in test case (mobilechain.js)

### DISCUSSION THREAD

**Discussion by Ashok-Sandhyala** [Shivamnema/Supply-Chain-Management-BlockChain-UseCase#11] [Q&A]
I would like to know how did you got that data part in test contract is it random or do we need to do anything to get that data...?
from mobilechain.js file

{
from:
**Data: '0x.............................'**
gas:
}

**Comment by Shivamnema** [comment]
I uses that to ease up the mining process of genesis block. This code is 4 years old and I think there might be some other way to resolve it. 
Why don't you take this opportunity to explore how can we start a local node privately using geth and create a PR.
